utils/links.js

```diff
@@ -0,0 +1,17 @@
+const links = [
+    { sizes: "57x57", href: "/apple-icon-57x57.png" },
+    { sizes: "60x60", href: "/apple-icon-60x60.png" },
+    { sizes: "72x72", href: "/apple-icon-72x72.png" },
+    { sizes: "76x76", href: "/apple-icon-76x76.png" },
+    { sizes: "114x114", href: "/apple-icon-114x114.png" },
+    { sizes: "120x120", href: "/apple-icon-120x120.png" },
+    { sizes: "144x144", href: "/apple-icon-144x144.png" },
+    { sizes: "152x152", href: "/apple-icon-152x152.png" },
+    { sizes: "180x180", href: "/apple-icon-180x180.png" },
+    { sizes: "192x192", href: "/android-icon-192x192.png" },
+    { sizes: "32x32", href: "/favicon-32x32.png" },
+    { sizes: "96x96", href: "/favicon-96x96.png" },
+    { sizes: "16x16", href: "/favicon-16x16.png" },
+];
+
+export {links};
```

Comment:
under utils make a folder `resources` so that in future if we add some utility functions then it should not be confusing to locate it.